src/go/k8s/tests/e2e-unstable/decommission/00-enable-downscaling.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
NIT: Why dcommision test is moved from stable e2e test to unstable?

Reply:
Because it can't, as it currently stands, be run in parallel. Will come back to it.